girder/api/sftp.py

```diff
@@ -261,32 +262,29 @@ def shutdown_request(self, request):
         pass
 
 
-def _main():  # pragma: no cover
+@click.command(name='sftpd', short_help='Run the Girder SFTP service.',
```

Comment:
Why are both `short_help` and `help` required? Would putting the help into a docstring help?